app/config/database.js

```diff
@@ -0,0 +1,50 @@
+const { DefaultAzureCredential } = require('@azure/identity')
+
+function isProd () {
+  return process.env.NODE_ENV === 'production'
+}
+
+const hooks = {
+  beforeConnect: async (cfg) => {
+    if (isProd()) {
+      const credential = new DefaultAzureCredential()
+      const accessToken = await credential.getToken('https://ossrdbms-aad.database.windows.net', { requestOptions: { timeout: 1000 } })
```

Comment:
I think we can remove or adjust this timeout as 1 second is quite low

Reply:
yes please